internal/cmd/operator-sdk/bundle/validate/cmd.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validate
+
+import (
+	"fmt"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/labels"
+
+	"github.com/operator-framework/operator-sdk/internal/cmd/operator-sdk/bundle/validate/internal"
+	"github.com/operator-framework/operator-sdk/internal/flags"
+)
+
+const (
+	longHelp = `The 'operator-sdk bundle validate' command can validate both content and format of an operator bundle
+image or an operator bundle directory on-disk containing operator metadata and manifests. This command will exit
+with an exit code of 1 if any validation errors arise, and 0 if only warnings arise or all validators pass.
+
+More information about operator bundles and metadata:
+https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
```

Comment:
IMO we ought to use the website links.

Reply:
@camilamacedo86 I don't see this design doc on the website anywhere. If there is a good description about them on the site, I will agree.

Reply:
There isn't a non-Github hosted link.